must-gather/collection-scripts/gather_logstore_resources

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
```

Comment:
same comment as above

Reply:
Resolving this comment in favour of eo-thread above.